src/css-grids/cssgrids.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <meta http-equiv="X-UA-Compatible" content="ie=edge">
+   <link rel="stylesheet" href="cssgrids.css">
+   <title>CSS Grids</title>
+</head>
+<body>
+   <div class="container">
+      <header>
+         <div>
```

Comment:
i feel like this div is doing a lot of stuff the header could do. (look at the css for it). unless there's a reason I can't see, remove div and give those styles to header.